packages/wallet/backend/tests/rafiki/controller.test.ts

```diff
@@ -0,0 +1,161 @@
+import { env } from "@/config/env";
+import { createContainer } from "@/createContainer";
+import { errorHandler } from "@/middleware/errorHandler";
+import { RafikiController } from "@/rafiki/controller";
+import { NextFunction, Request, Response } from "express";
+import {
+    MockRequest,
+    MockResponse,
+    createRequest,
+    createResponse,
+} from "node-mocks-http";
+import { Logger } from "winston";
+import {
+    mockGetRatesRequest,
+    mockOnWebhookRequest,
+    mockRafikiService,
+    mockRatesService,
+} from "../mocks";
+
+describe("Rafiki controller", () => {
+
+    let req: MockRequest<Request>;
+    let res: MockResponse<Response>;
+    let rafikiController: RafikiController;
+    let logger: Logger;
+
+    const next = jest.fn() as unknown as NextFunction;
+
+    const createRafikiControllerDepsMocked = () => {
+        const rafikiControllerDepsMocked = {
+            rafikiService: mockRafikiService,
+            logger,
+            ratesService: mockRatesService,
+        };
+        Reflect.set(rafikiController, "deps", rafikiControllerDepsMocked);
+    };
+
+    describe("Get Rates", () => {
+
+        beforeAll(async () => {
+            const container = createContainer(env);
+            rafikiController = await container.resolve("rafikiController");
+            logger = await container.resolve("logger");
+
+            createRafikiControllerDepsMocked();
+        });
+
+        beforeEach(async (): Promise<void> => {
+            req = createRequest();
+            res = createResponse();
+        });
+
+        it("should call method getRates() in ratesService", async () => {
+            const ratesSpy = jest.spyOn(mockRatesService, "getRates");
+            req.query = mockGetRatesRequest({ base: "USD" }).query;
+
+            await rafikiController.getRates(req, res, next);
+
+            expect(ratesSpy).toHaveBeenCalled();
+            expect(ratesSpy).toHaveBeenCalledTimes(1);
+            expect(ratesSpy).toHaveBeenCalledWith("USD");
+        }),
+
+            it("should return rates with base USD", async () => {
```

Comment:
there is lint issues I guess. can you please use command 'pnpm format'

Reply:
Alright, I will use this.